### ISSUE TITLE
Not report errors when there are errors in compiling bib file

### ISSUE THREAD

**Issue by fcyu** [SublimeText/LaTeXTools#703]
I found that it doesn't report errors when there are errors in bibtex,

**Comment by ig0774** [comment]
No. It doesn't claim to either. This could be a useful thing to add, however.

**Comment by fcyu** [comment]
It DOES helpful. There would be no reference in the PDF file and no error information in the console if the Latex succeed and bib failed. Even in the log file, there isn't any related information because it only contains the last Latex compiling output. The users would be confused unless they change `display_log` to `true`, and read the outputted log.

**Comment by fabrice102** [comment]
Nothing is displayed when texify fails because of bibtex.
It would be great that LaTeXTools indicates when texify fails even if it does not manage to parse the error log (e.g., because the error comes from bibtex).

For example, it could display "Unknown errors (return code: ...)" if it does not manage to find any error but the return code is not 0.
Even better, in that case, it could display the stderr output of the command.

**Comment by ig0774** [comment]
So, with `texify` the stderr output is... unhelpful to say the least. At best, you get a "texify did not succeed" message referring you to another log file that may or may not have helpful output from the LaTeX process.

Return codes are also a sticky issue as they are not consistently going to be `0` on success. That could probably be worked around, but then `latexmk` remaps all the `pdflatex` return codes and `texify` just returns `1` on an error, which means it still wouldn't be much more helpful than "something failed".

The right way around this is to parse the `.blg` files, if they exist, etc. This is made a little messy because 1) `biber` and `bibtex` don't agree on a log file format. And, 2) `bibtex`'s messages are very... inconsistent (e.g. `"I found no \citation commands---while reading file ..."`, `"Sorry, you've exceeded BibTeX's..."`). This isn't impossible to work with, it just means taking some time to figure out the possible messages and how to report them.

**Comment by deathaxe** [comment]
Build is terminated by LaTeXTools v4, in case any build command returns an error, regardless what log file contains.
